src/app/pages/ajustes/lista-movimientos/lista-movimientos.component.html

```diff
@@ -3,12 +3,12 @@
     <nb-card-header>
       Registrar Ajuste
     </nb-card-header>
-    <nb-card-body [nbSpinner]="!mostrar">
+    <nb-card-body [nbSpinner]="false">
```

Comment:
Si el valor es una constante, se puede omitir la parte dinámica
```suggestion
    <nb-card-body nbSpinner="false">
```

o directamente/mejor, eliminar el uso de la directiva
```suggestion
    <nb-card-body>
```